src/Mollie.Api/Client/ConnectClient.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Mollie.Api.Client {
     public class ConnectClient : BaseMollieClient, IConnectClient {
-        private const string AuthorizeEndPoint = "https://www.mollie.com/oauth2/authorize";
+        private const string AuthorizeEndPoint = "https://my.mollie.com/oauth2/authorize";
         private const string TokenEndPoint = "https://api.mollie.nl/oauth2/";
```

Comment:
@Viincenttt Ah shoot, I missed this - maybe worth double-checking why this is still mollie.nl and change to mollie.com?